scripts/team.coffee

```diff
@@ -0,0 +1,61 @@
+# Description:
+#   cluster lab members of a particular year randomly
+#   year and team size is mentioned in the command
+#   useful to randomly form teams in internal hackathons
+#
+# Configuration:
+#   INFO_SPREADSHEET_URL   
+#
+# Commands:
+#   listen for
```

Comment:
can replace this with `bot team [year] [team_size]`.
This way it will show properly in `bot help`.